runregistry-rest/rest.py

```diff
@@ -1,237 +1,20 @@
-                if file_ext not in app.config["UPLOAD_EXTENSIONS"]:
-                    error = "Unknown file extension! File needs to be .tar.gz or .tgz file! \n"
-                    return flask.make_response(error, 400)
-                local_file_name = os.path.join(app.config["UPLOAD_PATH"], filename)
-                if os.path.isfile(local_file_name):
-                    error = "BLOB insert is ongoing with the same file name! Try again a bit later."
-                    return flask.make_response(error, 400)
-                uploaded_file.save(local_file_name)
-            else:
-                error = "Expected file (conf blob) name is missing in form! \n"
-                return flask.make_response(error, 400)
-
-            # Read in file to memory
-            with open(local_file_name, "rb") as fin:
-                data = io.BytesIO(fin.read())
-
-            # Perform insert
-            query_list = []
-            bind_vars = []
-            query_list.append(queries.insertRunRegistryMeta)
-            query_list.append(queries.insertRunRegistryBlob)
-            bind_vars.append(
-                {
-                    "run_num": run_num,
-                    "det_id": det_id,
-                    "run_type": run_type,
-                    "filename": filename,
-                    "software_version": software_version,
-                }
-            )
-            bind_vars.append({"run_num": run_num, "config_blob": data.getvalue()})
-            db.perform_transaction_multi(query_list, bind_vars)
-            rowRes = []
-            db.perform_query(queries.getRunMeta, {"run_num": run_num}, rowRes)
-            resp = flask.make_response(flask.jsonify(rowRes))
-            # remove uploaded temp file
-            os.remove(local_file_name)
-            return resp
-        except Exception as e:
-            print("Exception:", e)
-            return flask.make_response(str(e), 400)
-
-
-@api.resource("/runregistry/updateStopTime/<int:runNum>")
-class updateStopTimestamp(Resource):
-    @auth.login_required
-    def get(self, runNum):
-        rowRes = []
-        try:
-            db.perform_transaction(queries.updateStopTime, {"run_num": runNum})
-            db.perform_query(queries.getRunMeta, {"run_num": runNum}, rowRes)
-        except Exception as e:
-            (err_obj,) = e.args
-            print("Exception:", str(e))
-            resp = flask.make_response(flask.jsonify({"Exception": err_obj.message}))
-            return resp
-        print(rowRes)
-        resp = flask.make_response(flask.jsonify(rowRes))
-        return resp
-
-
-@app.route("/")
-def index():
-    return "Best thing since sliced bread!"
-
+with app.app_context():
+    db.create_all()
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=5005, debug=True)
-
-# As a testserver.
-# app.run(host= '0.0.0.0', port=5000, debug=True)
-# Normally spawned by gunicorn
+    app.run(host="0.0.0.0", port=5005, debug=DEBUG)
```

Comment:
minor nit on missing new line at the end.

Reply:
added